documentation/docs/specs/did/iota_did_method_spec.md

```diff
@@ -312,49 +184,24 @@ The IOTA Identity framework currently supports two Verification Method Types:
 
 ### Revocation
 
-Revocation of Verifiable Credentials and signatures is currently achieved through revoking public keys in the IOTA Identity framework. Alternatively, developers should consider using the [`credentialStatus` property](https://www.w3.org/TR/vc-data-model/#status) when issuing and revoking Verifiable Credentials at scale.
+Revocation of Verifiable Credentials and signatures can be achieved using the [Revocation Bitmap 2022](../revocation_bitmap_2022.md) where issuers store a bitmap of indices in the DID Document. These indices correspond to verifiable credentials they have issued. If the binary value of the index in the bitmap is 1 (one), the verifiable credential is revoked, if it is 0 (zero) it is not revoked.
 
 ### Standardized Services
 
 The IOTA Identity framework also standardized certain `services` that are embedded in the DID Document. It is RECOMMENDED to implement these when implementing the `did:iota` method.
 
 Currently standardized `services`:
+
 * Nothing yet.
 
 ## Security Considerations
 
-The `did:iota` method is implemented on the [IOTA Tangle](https://iota.org), a public permissionless and feeless Distributed Ledger Technology (DLT), making it resistant against almost all censorship attack vectors. Up until the `Coordicide` update for the IOTA network, a reliability on the coordinator exists for resolving ordering conflicts. This has a minor censorship possibility, that, in the worst case, can prevent ordering conflicts from resolving to a DID. However, these can only be published by the owner of the private key and therefore does not constitute a usable attack vector. Lastly, a node may decide to censor DID messages locally, however a user can easily use another node. 
-
-Since DID messages are always to be signed and the 'chain of custody' of the signing key can be traced throughout the identity lifespan, replay, message insertion, deletion, modification and man-in-the-middle attacks are prevented.
```

Comment:
We really need a section on what aspects we are protected against by using the ledger. Notably how it deals with conflicts, replays, man-in-the-middle-attacks etc. The state index also provides a good linear history, if one has it.

We are not transparently cryptographically verifiable to clients anymore though (we rely on the protocol), so we might want to consider whether it's worth including some sort of signature on the DID Document payload as well.